.storybook/foundations/drop-shadow/index.css

```diff
@@ -0,0 +1,66 @@
+/*!
+Copyright 2024 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+@import "../../../components/commons/dropshadow.css";
+
+.spectrum-Foundations-Example-DropShadow-swatch {
+  block-size: 150px;
+  inline-size: 150px;
+  background-color: var(--spectrum-gray-25);
+  border-radius: var(--spectrum-corner-radius-large-default);
+  border: transparent;
+}
+
+.spectrum--dark .spectrum-Foundations-Example-DropShadow-swatch {
+  background-color: var(--spectrum-gray-75); /* matches dark mode design spec rgb(34, 34, 34) */
+}
+
+.spectrum-Foundations-Example-DropShadow-swatch--emphasized-default {
+  @extend %spectrum-DropShadow-emphasized-default;
+}
+
+.spectrum--dark .spectrum-Foundations-Example-DropShadow-swatch--emphasized-default {
+  /* override with correct color for dark mode in this example page only */
+  --mod-drop-shadow-emphasized-default-color: var(--spectrum-drop-shadow-color-100);
```

Comment:
Thanks for the helpful comment! Where were you able to find the dark mode color? I'm not seeing it in the design spec

Reply:
[The tokens design spec](https://www.figma.com/file/eoZHKJH9a3LJkHYCGt60Vb/S2-token-specs?type=design&node-id=14735-179&mode=design&t=xZO3TUNV9RHeV0ou-4) lists `drop-shadow-color-100` and `drop-shadow-color-200` token definitions, they're both black but the transparency percentages differ for each color token as well as light/dark.

I'm realizing on further inspection that the color token doesn't seem to be defined at all for light mode, so I'll look into why that's happening 🤔